modules/apps/object/object-admin-rest-api/src/main/resources/com/liferay/object/admin/rest/dto/v1_0/packageinfo

```diff
@@ -1 +1 @@
-version 6.1.1
\ No newline at end of file
+version 6.2.0
```

Comment:
Hey @p-albuquerque, could you move this change to the same commit as the previous comment?

Reply:
Done